examples/mnist_export_v2/export_test.py

```diff
@@ -0,0 +1,41 @@
+import tensorflow as tf
+import tensorflow_hub as hub
+import unittest
+import tempfile
+import os
+import re
+import export
+
+
+class TFHubMNISTTest(tf.test.TestCase):
+    def setUp(self):
+        self.mock_dataset = tf.data.Dataset.range(5)
+        self.mock_dataset = self.mock_dataset.map(
```

Comment:
Please chain the dataset expressions together.

Reply:
Done